### DISCUSSION TITLE
Node chronos-repeat

### DISCUSSION THREAD

**Discussion by ghost** [jensrossbach/node-red-contrib-chronos#70] [Q&A] [answered]
Hi,
I try to explain .....
I use some node repeat (for example every 10 min) and when the command is "msg.payload = true" I have immediately the first event in output when in fact 10 min are not gone; I think that the first output should be after 10 min.
How can I fix it?

**Comment by jensrossbach** [comment, marked as answer]
Hi, since release 1.15 you have the possibility to fine tune the sending behavior upon message ingress. Just change the option "When message arrives on input" to "do nothing" and you should get your requested behavior.

**Comment by ghost** [comment]
Ok, thank you very much .....
